File: Python/FizzBuzz.py
# file: FizzBuzz.py
#
# author: Josh Grant 2016
#
# FizzBuzz (with arguments) in Python 2.7.10

from sys import argv

script, first, last = argv

first = int(first)
last = int(last) + 1

for i in range(first, last):
    if (i % 3 == 0) and (i % 5 != 0):
        print "Fizz"
    elif (i % 3 != 0) and (i % 5 == 0):
        print "Buzz"
    elif (i % 15 == 0):
        print "FizzBuzz"
    else:
        print i

File: NodeJS/FizzBuzz.js
// file: FizzBuzz.js
//
// author: Josh Grant 2016
//
// FizzBuzz in JavaScript (NodeJS)

var strOut, i;

for (i = process.argv[2]; i <= process.argv[3]; i++) {
    strOut = "";
    if (i % 3 === 0) {
        strOut += "Fizz";
    }
    if (i % 5 === 0) {
        strOut += "Buzz";
    }
    if (strOut === "") {
        strOut = i;
    }
    console.log(strOut);
}